content/ops/creating-a-local-dev-environment-in-Virtual-Box.md

```diff
@@ -196,8 +196,6 @@ Secure copy the cflinuxfs2 container to the CF VM.
 
 Connect using the Cloud Foundry CLI.
 
-	# Run this from inside your Vagrant Bosh Lite instance
-	vagrant ssh
```

Comment:
It was a mistake before - there was something which caused my DNS not to resolve bosh-lite.com, causing the Vagrant host not to work. After a full repeat of the initial instructions, I found it was no longer needed. May have to do with VPN but I haven't reproduced it.